src/tests/unit/tests/popup/components/diagnostic-view-toggle.test.tsx

```diff
@@ -226,13 +230,14 @@ describe('DiagnosticViewToggleTest', () => {
 
             const props: DiagnosticViewToggleProps = propsBuilder.build();
 
-            const component = React.createElement(DiagnosticViewToggle, props);
+            const component = React.createElement(TestDiagnosticViewToggle, props);
+
+            const wrapper = render(component);
+            const toggle = wrapper.getByRole('switch');
+            fireEvent.focus(toggle);
```

Comment:
We shouldn't need the `fireEvent` or `rerender` calls here since our new class is setting the properties for initial render.

Reply:
removed